### PR TITLE
Upgrade to K8s 1.19

### PR DESCRIPTION
**Pull request by umangachapagain**
Updates dependencies & Fixes broken changes

- Kubernetes -> v0.19.3 (OCP 4.6)
- Controller Runtime -> v0.6.3

This forms the base for moving to SDK v1.2.0 (Updated code-generation tools and project structure)

Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

### CODE REVIEW

File: pkg/deploy-manager/subscription.go [hunk 1]
```diff
@@ -40,16 +41,16 @@ func (t *DeployManager) deployClusterObjects(co *clusterObjects) error {
...
-			return err
-		}
+	// for _, operatorGroup := range co.operatorGroups {
+	// 	_, err := t.olmClient.OperatorsV1()
+	// 	if err != nil && !errors.IsAlreadyExists(err) {
+	// 		return err
+	// 	}
 
-	}
+	// }
```
Comment (jarrpa) [inline pkg/deploy-manager/subscription.go:50]: Remove these comments.

File: cmd/manager/main.go [hunk 2]
```diff
@@ -64,7 +61,8 @@ func main() {
 	printVersion()
 	log.Info(fmt.Sprintf("Running in development mode: %v", isDevelopmentEnv))
 
-	namespace, err := k8sutil.GetWatchNamespace()
+	// TODO: Remove once migrated to new scafold
```
Comment (jarrpa) [inline cmd/manager/main.go:64]: What does this mean? What is "new scaffold"? Please be specific in the comment, provide links if needed.

**Review by jarrpa — CHANGES_REQUESTED** [review]

File: pkg/deploy-manager/subscription.go (hunk 1 above)
Comment (umangachapagain) [reply pkg/deploy-manager/subscription.go]: Fixed.

File: cmd/manager/main.go (hunk 2 above)
Comment (umangachapagain) [reply cmd/manager/main.go]: Done.

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**